CodeAssignment_MVVM_Combine/Sources/Views/LoadingView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  LoadingView.swift
+//  CodeAssignment_MVVM_Combine
+//
+//  Created by gyusoku.i on 2024/05/27.
+//
+
+import UIKit
+import SnapKit
+
+final class LoadingView: UIView {
+    private let backgroundView: UIView = {
+        let view = UIView()
+        view.backgroundColor = UIColor.black.withAlphaComponent(0.12)
+        return view
+    }()
+    
+    private lazy var loadingIndicatorView: UIActivityIndicatorView = {
+        let indicator = UIActivityIndicatorView(style: .large)
+        indicator.color = .systemGray
+        indicator.hidesWhenStopped = true
+        return indicator
+    }()
+    
+    private lazy var searchingLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 16, weight: .semibold)
+        label.text = "検索中..."
+        label.textAlignment = .center
+        label.textColor = .systemGray
+        return label
+    }()
+    
+    var isLoading = false {
+        didSet {
+            self.isHidden = !self.isLoading
+            self.isLoading ? loadingIndicatorView.startAnimating() : loadingIndicatorView.stopAnimating()
```

Comment:
zozotown-iosでは、selfは必要ない時は基本的には付けない方針なのでこう書いてます。

```suggestion
            isHidden = !isLoading
            isLoading ? loadingIndicatorView.startAnimating() : loadingIndicatorView.stopAnimating()
```

Reply:
[821f8c7](https://github.com/KyusokLee/CodeAssignment_MVVM_Combine/pull/7/commits/821f8c72a442931b36d8ec581c56d55447187f96)にてコード修正しました！ご確認お願いします！